client/app/lib/components/table/MuiTableAdapter/MuiTableToolbar.tsx

```diff
@@ -19,14 +19,14 @@ const ToolbarContainer = ({ children }: ToolbarContainerProps): JSX.Element => (
 const MuiTableToolbar = (props: ToolbarProps): JSX.Element | null => {
   const { t } = useTranslation();
 
-  if (props.alternative?.when())
-    return <ToolbarContainer>{props.alternative.render()}</ToolbarContainer>;
-
-  if (props.renderNative)
-    return (
-      <ToolbarContainer>
-        <div className="flex w-full justify-between">
-          {props.onSearchKeywordChange && (
+  if (!props.alternative?.when() && !props.renderNative) return null;
+
+  return (
+    <ToolbarContainer>
+      <div className="flex w-full justify-between">
+        {(props.renderNative ||
+          (props.alternative?.when() && props.alternative.keepNative)) &&
```

Comment:
This has modified the behaviour of `ManageUsersTable`

Reply:
done. Apparently this `renderNative` is always true whenever `toolbar` inside table is defined, and the value `show` is set to `true`. I have modified it accordingly